app/cli/internal/action/attestation_status.go

```diff
@@ -150,6 +138,61 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 	return res, nil
 }
 
+// populateContractMaterials populates the materials that are defined in the contract schema
+func (action *AttestationStatus) populateContractMaterials(res *AttestationStatusResult, visitedMaterials map[string]struct{}) error {
+	for _, m := range action.c.CraftingState.InputSchema.Materials {
+		materialResult := &AttestationStatusResultMaterial{
+			Material: &Material{
+				Name: m.Name, Type: m.Type.String(),
+				Annotations: pbAnnotationsToAction(m.Annotations),
+			},
+			IsOutput: m.Output, Required: !m.Optional,
+		}
+
+		if cm, found := action.c.CraftingState.Attestation.Materials[m.Name]; found {
+			if err := setMaterialValue(cm, materialResult.Material); err != nil {
+				return err
+			}
+			materialResult.Set = true
+			materialResult.Tag = cm.GetContainerImage().GetTag()
+		}
+
+		res.Materials = append(res.Materials, *materialResult)
+		visitedMaterials[m.Name] = struct{}{}
+	}
+	return nil
+}
+
+// populateAdditionalMaterials populates the materials that are not defined in the contract schema
+func (action *AttestationStatus) populateAdditionalMaterials(res *AttestationStatusResult, visitedMaterials map[string]struct{}) error {
+	for name, m := range action.c.CraftingState.Attestation.Materials {
+		if _, found := visitedMaterials[name]; found {
+			continue
+		}
+
+		// No need to check for name collisions, as it is not defined in the contract schema and it's
+		// autogenerated by the crafter
+		materialResult := &AttestationStatusResultMaterial{
+			Material: &Material{
+				Name:        name,
+				Type:        m.GetMaterialType().String(),
+				Annotations: stateAnnotationToAction(m.Annotations),
+			},
+			// No need to check if the material is optional or not, as it is not defined in the contract schema
+			IsOutput: true, Required: false,
```

Comment:
add a todo saying that `isOutput` this will be configurable someday

Out of curiosity, why did you choose to make this output `true`?

Reply:
> Out of curiosity, why did you choose to make this output true?
I thought we might wanted them to be shown in the final graph however, thinking now I think that's the default behavior right?

Shall we set it to false?

Reply:
TBH I do not know, it's probably sensible to stick to the `false` default.

In any case I believe this should be configurable in the future, probably passing a flag. Could you create a follow up task for that?

Reply:
Sure, task created and field changed.